src/Actions/StubResolver/CopyStubAction.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace KoalaFacade\DiamondConsole\Actions\StubResolver;
+
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Str;
+use KoalaFacade\DiamondConsole\Foundation\Action;
+
+class CopyStubAction extends Action
+{
+    /**
+     * Copy Stub to Diamond Infrastructure
+     *
+     * @param  string  $stubPath
+     * @param  string  $destinationPath
+     * @param  string  $name
+     * @param  array<string>  $replacements
+     * @return void
+     */
+    public function execute($stubPath, $destinationPath, $name, $replacements): void
```

Comment:
i think name is not obvious for this action, let say i consume this action in another file, its like make me confuse what is exactly name purpose? 

I suggest using `$fileName` if you had better naming can put it also